editor/selectors.js

```diff
@@ -736,7 +736,7 @@ export function getBlockFocus( state, uid ) {
  * @return {Boolean}      True if multi-selecting, false if not.
  */
 export function isMultiSelecting( state ) {
-	return !! state.blockSelection.isMultiSelecting;
+	return state.blockSelection.isMultiSelecting;
```

Comment:
See my comment on `editor/reducers` about refactoring `blockSelection`: since `isMultiSelecting` isn't its own reducer, but just a field of `blockSelection`, part of me would feel safer with using `lodash#get` here.